services/src/main/scala/cromwell/services/metadata/MetadataService.scala

```diff
@@ -129,10 +131,12 @@ object MetadataService {
   def throwableToMetadataEvents(metadataKey: MetadataKey, t: Throwable): List[MetadataEvent] = {
     t match {
       case aggregation: ThrowableAggregation =>
-        aggregation.errorMessages.toList map { message => MetadataEvent(metadataKey.copy(key = s"${metadataKey.key}:message"), MetadataValue(s"${aggregation.exceptionContext}: $message")) }
+        val message = List(MetadataEvent(metadataKey.copy(key = s"${metadataKey.key}[${Random.nextInt(Int.MaxValue)}]:message"), MetadataValue(aggregation.exceptionContext)))
```

Comment:
I realize this is somewhat a refactoring of what was here previously, but the space of nonnegative `Int`s is kind of limited for a hash...